task/02-numbers-tasks.js

```diff
@@ -221,7 +228,7 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+    return parseFloat(value) ? parseFloat(value) : def;
```

Comment:
still not working, you need check is `value` a Number, `parseFloat('0')` is interpreted as false and return `def`
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN